src/components/line-tabs/styles/SLineTabs.tsx

```diff
@@ -0,0 +1,55 @@
+import styled, { css } from 'styled-components';
+import { CSSProgressiveBody03Semibold } from '../../../constants/styles/fonts';
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from '../../../constants/styles/mediaquerys';
+import { ILineTabsProps } from '../LineTabs';
+
+const CSSHover = css`
+  color: var(--hover-text-04);
+  border-bottom: 1px solid var(--hover-text-04);
+`;
+
+const CSSActivated = css`
+  color: var(--text-01);
+  border-bottom: 1px solid var(--text-01);
+`;
+
+const Bronze = css<ILineTabsProps>`
+  ${CSSProgressiveBody03Semibold};
+  color: var(--text-04);
+  max-width: min-content;
+  white-space: nowrap;
+  padding-bottom: 15px;
+  display: flex;
+  align-items: center;
+  ${(props) => props.hover ? CSSHover : ''};
```

Comment:
why the CSSHover is applied directly instead of using the pseudo class?

Reply:
This is to set and change the state of the component (coming from the design file: hover and activated) also with the storybook controls.